ArmoniK.Extension.CSharp.Client/Common/Domain/ApplicationConfiguration.cs

```diff
@@ -16,11 +16,33 @@
 
 namespace ArmoniK.Extension.CSharp.Client.Common.Domain;
```

Comment:
IMO, this should be part of the worker associated extensions

Reply:
I agree, I'll remove this class.